puppeteer.setup.js

```diff
@@ -1,9 +1,18 @@
 const puppeteer = require('puppeteer');
 
+global.stepNumber = 10;
```

Comment:
currently, it take long time. let's reduce the number. 5?

Reply:
I agree